### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by parispittman**
added a contributing header to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,4 +29,26 @@ Add the dependency to any targets you've declared in your manifest:
 .target(name: "MyTarget", dependencies: ["SymbolKit"]),
 ```
 
+
+## Contributing to Symbolkit
+
+Contributions to /swift-docc-symbolkit are welcomed and encouraged! Please see the
+[Contributing to Swift guide](https://swift.org/contributing/).
```
Comment (QuietMisdreavus) [inline README.md:36]: SymbolKit's library name is `SymbolKit`, capitalized:

```suggestion
## Contributing to SymbolKit

Contributions to SymbolKit are welcomed and encouraged! Please see the
[Contributing to Swift guide](https://swift.org/contributing/).
```

**Comment by shahmishal** [comment]
@swift-ci test

File: README.md [hunk 2]
```diff
@@ -29,4 +29,26 @@ Add the dependency to any targets you've declared in your manifest:
...
+
+## Contributing to SymbolKit
+
+Contributions to SymbolKit are welcomed and encouraged! Please see the
+[Contributing to Swift guide](https://swift.org/contributing/).
+
+Before submitting the pull request, please make sure you have [tested your
+ changes](https://github.com/apple/swift/blob/main/docs/ContinuousIntegration.md)
+ and that they follow the Swift project [guidelines for contributing
+ code](https://swift.org/contributing/#contributing-code).
```
Comment (QuietMisdreavus) [inline README.md:41]: looks like some weird formatting here:

```suggestion
Before submitting the pull request, please make sure you have [tested your
changes](https://github.com/apple/swift/blob/main/docs/ContinuousIntegration.md)
and that they follow the Swift project [guidelines for contributing
code](https://swift.org/contributing/#contributing-code).
```

**Comment by QuietMisdreavus** [comment]
@swift-ci Please test

**Review by QuietMisdreavus — APPROVED** [review]

**Pull request merged**